assets/js/mermaid/index.js

```diff
@@ -0,0 +1,31 @@
+let idCount = 0;
```

Comment:
I'd move this file to `js/cell/markdown/mermaid.js` as it's specific to our markdown :)

Reply:
should `js/cell/markdown.js` be moved to `js/cell/markdown/index.js`? That's just my assumption based on how the rest of the JS looks organized

Reply:
Most of the directories are individual LV hooks and in those we have `index.js`, other than that we use regular file names :)